fedora/jellyfin-web.spec

```diff
@@ -2,11 +2,11 @@
 
 Name:           jellyfin-web
 Version:        10.8.0
-Release:        1%{?dist}
+Release:        2%{?dist}
 Summary:        The Free Software Media System web client
 License:        GPLv3
 URL:            https://jellyfin.org
-# Jellyfin Server tarball created by `make -f .copr/Makefile srpm`, real URL ends with `v%{version}.tar.gz`
+# Jellyfin Server tarball created by `make -f .copr/Makefile srpm`, real URL ends with `v%%{version}.tar.gz`
```

Comment:
Is `%%` here intentional?

Reply:
Not certain but it's a comment.

Reply:
cc @brianjmurrell because I'm not sure

Reply:
Yes, `%%` is intentional.  Macros are not supposed to be used in comments, so the double-`%` escapes the `%` so that rpmbuild no longer sees it as a macro.

Reply:
FWIW, I am happy to have a webhook installed for this repo (like I do already for the jellyfin/jellyfin repo) so that my COPR project automatically builds all tags, and we can get on top of these issues while they are still in alpha/beta/rc/etc.

That can be done with:
1. In your GitHub project, go to Settings / Webhooks.
1. Click on the Add webhook button.
1. Fill in the Payload URL field with the url I can send you by PM.
1. Select application/json as the content type.
1. Click the Add webhook button.

Reply:
That's not really true.

https://copr.fedorainfracloud.org/coprs/brianjmurrell/jellyfin/package/jellyfin/ is subscribed to a webhook (courtesy of @joshuaboniface) and builds all jellyfin tags.

I'm not even sure why you are resisting this.  This is a contribution to your project.  It provides mock-root built RPMs.  That provides testing of your RPM builds as well as provides RPMs to your users.

@joshuaboniface Are you really not interested in having early and automated builds of jellyfin-web, as you have with jellyfin-server?

Reply:
The keyword was "avoid", to which @dkanada's comment isn't _wrong_. I forget if we've already communicated through other channels or not, but send me an e-mail and we'll see about getting things sorted out.